components/portal/main/services.js

```diff
@@ -119,10 +122,30 @@ define(['angular'], function(angular) {
         return windowTitle;
       }
 
+      /**
+       * Get banner messages from bannersURL endpoint
+       */
+      function getBanners() {
+        return $http.get(SERVICE_LOC.bannersURL)
+          .then(function(response) {
+            if (response.data && response.data.messages
+              && angular.isArray(response.data.messages)) {
+              return response.data.messages;
+            } else {
+              return GET_BANNERS_FAILED;
+            }
+          })
+          .catch(function(error) {
+            miscService.redirectUser(error.status, 'Get all messages');
```

Comment:
Should `'Get all messages'` here be something more banner-flavored so as to not conflate this error redirection case with an error related to getting (bell  notification) messages?